preprocess/SortVcf.py

```diff
@@ -171,62 +171,57 @@ def sort_vcf_from(args):
     else:
         output = open(output_fn, 'w')
 
-    for contig in contigs_order_list:
-        contig_dict = defaultdict(str)
-        contig_vcf_fns = [fn for fn in all_files if contig in fn]
-        for vcf_fn in contig_vcf_fns:
-            file = os.path.join(input_dir, vcf_fn)
-            if is_lz4_format:
-                read_proc = subprocess_popen(shlex.split("{} {}".format("lz4 -fdc", file)), stderr=subprocess.DEVNULL)
-                fn = read_proc.stdout
-            else:
-                fn = open(file, 'r')
-            for row in fn:
-                row_count += 1
-                if row[0] == '#':
-                    # skip phasing command line only occur with --enable_phasing, otherwise would lead to hap.py evaluation failure
-                    if row.startswith('##commandline='):
-                        continue
-                    if row not in header:
-                        header.append(row)
+    contig_dict = defaultdict(str)
+    for vcf_fn in all_files:
```

Comment:
The old sorting relied on vcfs created with the contig names in them. This new process has the same result but the VCF files can be named anything

Reply:
The old sorting is required as it will not maintain a contig_dict with too many records, we noticed that without contig spliting, it would increase the memory in sort_vcf submodule significantly

Reply:
Hey @zhengzhenxian , I'm curious why you merged this pull request if this sorting method will use too much memory. Should I create another pull request to change this to be more similar to the old method but also handle the multi-contig vcf the new code path creates?

Reply:
I have reverted the code with the previous sorting logic after merging the PR.